Makefile

```diff
@@ -62,8 +62,8 @@ win_xcalib: xcalib.c
 	$(CC) $(CFLAGS) -mwindows -lm resource.o -o xcalib xcalib.o
 
 install:
-	cp ./xcalib $(DESTDIR)/usr/local/bin/
-	chmod 0644 $(DESTDIR)/usr/local/bin/xcalib
+	cp -pv ./xcalib $(DESTDIR)/usr/local/bin/
```

Comment:
What is the -pv used for? Is that portable? If not, please skip.

Reply:
the -pv is to make a nice message for the installer and preserve the builder's ownership. it's in the cp of GNU coreutils, so yes, i think it's portable. do what you think is appropriate, of course.

Reply:
Thanks. A GNU extension might be no good fit for various X11 operating systems. Please omit for a merge.